### PR TITLE
Update dependencies and pom file.

### PR DESCRIPTION
**Pull request by waikontse**
Updated dependencies/pom to work with a newer JDK

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -38,6 +36,9 @@
                 <plugin>
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>${maven-compiler-plugin.version}</version>
+                    <configuration>
+                        <release>${release.version}</release>
```
Comment (mout-rabobank) [inline pom.xml:40]: The -release argument for the Java compiler is supported in Java 9+, but we still need to use Java 8 here.

File: sdk/pom.xml [hunk 2]
```diff
@@ -121,12 +123,19 @@
...
+                <configuration>
+                    <release>8</release>
+                </configuration>
+            </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${maven-surefire-plugin.version}</version>
                 <configuration>
-                    <argLine>-Duser.timezone=Europe/Amsterdam ${surefireArgLine}</argLine>
```
Comment (mout-rabobank) [inline sdk/pom.xml:129]: ${surefireArgLine} must kept for measuring the coverage.

File: sdk/pom.xml [hunk 3]
```diff
@@ -3,10 +3,15 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>nl.rabobank.oss.omnikassa</groupId>
+    <parent>
+        <groupId>nl.rabobank.oss.omnikassa</groupId>
+        <artifactId>java-sdk-parent</artifactId>
+        <version>${revision}${sha1}</version>
+    </parent>
```
Comment (mout-rabobank) [inline sdk/pom.xml:10]: Let's discuss this.

**Review by mout-rabobank — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (waikontse) [reply pom.xml:40]: True, it's bit more complicated according to this post.
https://stackoverflow.com/questions/59049980/maven-compiler-release-as-an-replacement-for-source-and-target

Maybe provide official releases targeted for Java8?

File: pom.xml [hunk 4]
```diff
@@ -13,14 +13,12 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <java.version>1.8</java.version>
-        <maven.compiler.source>${java.version}</maven.compiler.source>
-        <maven.compiler.target>${java.version}</maven.compiler.target>
-
-        <maven-war-plugin.version>2.6</maven-war-plugin.version>
-        <maven-assembly-plugin.version>2.6</maven-assembly-plugin.version>
-        <maven-compiler-plugin.version>3.5.1</maven-compiler-plugin.version>
+        <release.version>8</release.version>
```
Comment (mout-rabobank) [inline pom.xml:16]: Please revert to using maven.compiler.source + maven.compiler.target

**Review by mout-rabobank — APPROVED** [review]

**Pull request merged**